### DISCUSSION TITLE
Where is `useExportType`?

### DISCUSSION THREAD

**Discussion by goustaff** [biomejs/biome#1156] [Q&A] [answered]
Hello!
I would like to use `useExportType`... But...

- It is documented as being in the `style` group here: https://biomejs.dev/linter/rules/#style
- It is documented as being in the `nursery` group here: https://biomejs.dev/linter/rules/use-export-type/
- I can't find it anywhere, nor in https://biomejs.dev/schemas/1.4.1/schema.json, nor in https://biomejs.dev/schemas/1.4.0/schema.json

Thank you for your answers!

EDIT: [useGroupedTypeImport](https://biomejs.dev/linter/rules/use-grouped-type-import) is also documented as `style` here: https://biomejs.dev/linter/rules, while it is in the `nursery` group.

**Comment by Conaclos** [comment, marked as answer]
Hi!

The rule is not yet released. According to the [documentation](https://biomejs.dev/linter/rules/use-export-type/) the rule is in `vnext` version. Also, you can notice that the rule is in the `Unreleased` section of [our changelog](https://github.com/biomejs/biome/blob/main/CHANGELOG.md#unreleased).

I don't see any issue with the category, both rules are in nursery.

EDIT: I think the issue is coming from the use of an URL anchor that highlights `style` while you are currently looking at the nursery group?

**Reply by goustaff** [reply]
> EDIT: I think the issue is coming from the use of an URL anchor that highlights style while you are currently looking at the nursery group?

Oh, maybe!
I noticed that the highlighting was sometimes confusing after posting this. Sorry.

**Reply by Conaclos** [reply]
I was confused too 😅

**Comment by Conaclos** [comment]
@gustaveWPM You can now try the rule using our [latest nightly](https://github.com/biomejs/biome/releases/tag/cli%2Fv1.4.1-nightly.570d680).

**Reply by goustaff** [reply]
Oh, thank you a lot!
Gonna try it this afternoon. :)
